app/workers/rating_votes/assign_rating_worker.rb

```diff
@@ -0,0 +1,21 @@
+module RatingVotes
+  class AssignRatingWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :low_priority, retry: 10
+
+    def perform(article_id, group = "experience_level")
+      article = Article.find(article_id)
```

Comment:
this will fail if the article_id is not found, it won't go to the following line. `find_by(id: )` will work

Reply:
✅